dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt

```diff
@@ -65,7 +65,7 @@ class MetaDataControllerTest {
         for ((company, data) in testData) {
             val testCompanyId = companyDataControllerApi.postCompany(company).companyId
             data.forEach {
-                euTaxonomyDataForNonFinancialsControllerApi.postCompanyAssociatedData(
+                euTaxonomyDataForNonFinancialsControllerApi.postCompanyAssociatedData1(
```

Comment:
Would it make sense to give this another name, one that is more speaking then postCompanyAssociateData1? Like postCompanyAssociateDataTaxonomyNonFinancial or is this not possible?

Reply:
Already changed.